src/parseTools_legacy.js

```diff
@@ -110,6 +117,7 @@ global.Runtime = {
 global.ATMAINS = [];
 
 function addAtMain(code) {
+  warn('use of legacy parseTools function: addAtMain');
```

Comment:
Would it be reasonable to add an actionable recommendation to these warnings? Are there newer versions of all these functions that users could switch to?

Reply:
That would require more work on a per-symbol basis.   The likelyhood of external code actually using these functions is very very low so I'm not too worried about it.  

My rational for adding these warnings is that in a year or so we can probably completely remove them.